app/src/main/java/com/jerboa/MainActivity.kt

```diff
@@ -133,7 +133,7 @@ class MainActivity : AppCompatActivity() {
                     appSettings.usePrivateTabs,
                 )
 
-                ShowChangelog(appSettingsViewModel = appSettingsViewModel)
+                ShowChangelog()
```

Comment:
Stuff like this scares me, it seems like it has no actual dependencies now, when in reality the appSettings does need to be injected.